src/flamenco/rewards/fd_rewards.c

```diff
@@ -660,15 +650,6 @@ calculate_validator_rewards( fd_bank_t *                    bank,
   /* If there are no points, then we set the rewards to 0. */
   *rewards_out = points>0UL ? *rewards_out: 0UL;
 
-  if( capture_ctx ) {
-    ulong const epoch = fd_bank_epoch_get( bank );
-    fd_solcap_writer_stake_rewards_begin( capture_ctx->capture,
-        epoch,
-        epoch-1UL, /* FIXME: this is not strictly correct */
-        *rewards_out,
-        (fd_w_u128_t){ .ud=points } );
```

Comment:
Solcap v2 should only be merged when it's at feature parity with solcap v1, if solcap v1 is being simultaneously removed.